tests/model_validation/tests.py

```diff
@@ -1,20 +1,28 @@
 from django.core import management
-from django.core.checks import Error, run_checks
+from django.core.checks import Error
+from django.core.checks.model_checks import _check_lazy_references
 from django.db.models.signals import post_init
 from django.test import SimpleTestCase
-from django.test.utils import override_settings
+from django.test.utils import isolate_apps, override_settings
 from django.utils import six
 
 
 class OnPostInit(object):
     def __call__(self, **kwargs):
         pass
 
+    def on_post_init(self):
+        pass
+
 
 def on_post_init(**kwargs):
     pass
 
 
+def print_model(model):
```

Comment:
I'm unsure about the purpose of this. No tests fail if I change `print(model)` to `pass` and print make me nervous that something might go stdout (which doesn't seem to be the case) but at least some comment would help.